src/db/MediaObjectTypes.ts

```diff
@@ -1,11 +1,11 @@
-import { ObjectId } from 'mongoose'
+import mongoose from 'mongoose'
 import { MUUID } from 'uuid-mongodb'
 import { Point } from '@turf/helpers'
 
 export type ImageFormatType = 'jpeg' | 'png' | 'webp' | 'avif'
 
 export interface MediaObject {
-  _id: ObjectId
+  _id: mongoose.Types.ObjectId
```

Comment:
we've been using ObjectId type incorrectly.  

- `mongoose.Types.ObjectId` is the typescript type and also the class to generate new mongo object id
- `mongoose.ObjectId` should only be used for describing the schema